var/spack/repos/builtin/packages/easybuild/package.py

```diff
@@ -12,10 +12,16 @@ class Easybuild(PythonPackage):
     """
 
     homepage = 'http://hpcugent.github.io/easybuild/'
```

Comment:
This should be updated to https://easybuilders.github.io/easybuild (the old link auto-redirects, but might as well get it right).